streamer/etc/pulp/streamer.conf

```diff
@@ -0,0 +1,27 @@
+# ===========================
+# Pulp Streamer Configuration
+# ===========================
+
+# Manages settings for the streamer component of the lazy loading feature
+# for Pulp.
+#
+# port: the port that the Pulp Streamer listens on for incoming requests.
+#     This should match the port that Squid is configured to forward to.
+#
+# interfaces: comma-separated list; the interfaces that the Pulp Streamer
+#     listens on. These should be either host names or IP addresses. The
+#     Pulp Streamer defaults to listening on the 'localhost' interface.
```

Comment:
'localhost' is a hostname. The corresponding network interface is usually named ``lo``. Which do you actually want here?